src/plone/restapi/services/actions/get.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from Products.CMFCore.interfaces import IActionCategory
+from Products.CMFCore.utils import getToolByName
+from plone import api
+from plone.restapi.interfaces import IExpandableElement
+from plone.restapi.services import Service
+from zope.component import adapter
+from zope.i18n import translate
+from zope.interface import Interface
+from zope.interface import implementer
+
+
+@implementer(IExpandableElement)
+@adapter(Interface, Interface)
+class Actions(object):
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+    def __call__(self, expand=False):
+        result = {
+            'actions': {
+                '@id': '{}/@actions'.format(self.context.absolute_url()),
+            },
+        }
+        if not expand:
+            return result
+
+        context_state = api.content.get_view(
```

Comment:
Are we finally allowing plone.api in the code?

Reply:
No. @csenger could you fix that please?